src/app_service/common/json_utils.nim

```diff
@@ -72,3 +73,11 @@ template getProp(obj: JsonNode, prop: string, value: var typedesc[Address]): boo
     success = true
 
   success
+
+template getProp(obj: JsonNode, prop: string, value: var typedesc[MemberRole]): bool =
+  var success = false
```

Comment:
minor: `result` variable can be used instead of custom `success`

Reply:
It has the same code stile as in other similar fn in this file

Reply:
no problem 🙂